gulpfile.js

```diff
@@ -0,0 +1,90 @@
+var gulp = require('gulp');
```

Comment:
This gulpfile doesn't transpile ES6 codes into ES5. Please do so into follow directories: dist-client, dist-server, dist-test.